internal/bminventory/inventory.go

```diff
@@ -1222,6 +1222,16 @@ func (b *bareMetalInventory) InstallClusterInternal(ctx context.Context, params
 		return nil, common.NewApiError(http.StatusNotFound, err)
 	}
 
+	err = b.db.Model(&models.Cluster{}).Where("id = ?", cluster.ID.String()).Updates(&models.Cluster{
```

Comment:
what will be the side effect if it will fail?

Reply:
The effect would be that the user finds themselves having to restart the installation twice. Maybe we should hard fail if this field cannot be updated?

Reply:
yes it should be a hard failure

Reply:
OK, hard failure added for this case.